phoneApp/js/editContact.js

```diff
@@ -3,59 +3,49 @@
 //- изменять backgroundColor
 
 class EditContact {
-  constructor(globalState){
-    this.state = globalState; //стал равен this.state-у со страницы App.js
-  	this.phoneNumber = '+38 (063) 733 44 55';
-  }
+	constructor(globalState) {
+		this.state = globalState; //стал равен this.state-у со страницы App.js
+		this.phoneNumber = "+38 (063) 733 44 55";
+	}
 
-  renderInfo(value) {
-
-  	return `<div class="edit-field">
+	renderInfo(value) {
+		return `<div class="edit-field">
 				<button href="#" class="add-btn"><span class="glyphicon glyphicon-plus-sign" aria-hidden="true"></span>
 					<input type="text" placeholder="${value}"></input>
 				</button>
 			</div>`;
-  }
-
-  renderLink(options) {
-    let {href, glyphicon, text, active} = options;
-    let activeClass = active ? 'active' : '';
-
-    return `<a href="${href}.html" class="tab ${activeClass}">
-              <span class="glyphicon glyphicon-${glyphicon}" aria-hidden="true"></span>
-              <span class = "tab-text">${text}</span>
-            </a> `
-  }
-
-  buttonsHandler(){
-  	let buttonsParent = document.querySelector('main');
-  	buttonsParent.addEventListener('click', this.clickHandler.bind(this));
-  }
-
-  clickHandler(e) {
-
-  	let target = e && e.target;
-  	if (!target) return;
-
-  	let active = e && e.target && (e.target.closest('button') || e.target.classList.contains('add-btn'));
- 	if (active == false) return;
-
- 	let input = active.querySelector('input');
- 	input.style.backgroundColor = 'lightgreen';
-
- 	input.addEventListener('blur', () => {
- 		input.removeAttribute('style');
- 	})
-
-  }
-
-
-  setEvents(){
-  	this.buttonsHandler();
-  }
-
-  render() {
-  	return `<header class="header">
+	}
+
+	buttonsHandler() {
+		let buttonsParent = document.querySelector("main");
+		buttonsParent.addEventListener("click", this.clickHandler.bind(this));
+	}
+
+	clickHandler(e) {
+		let target = e && e.target;
+		if (!target) return;
+
+		let active =
+			e &&
+			e.target &&
+			(e.target.closest("button") ||
+				e.target.classList.contains("add-btn"));
```

Comment:
what if another dev would change that className at button - javascript would break ?
We shouldn't rely on classsNames